velox/exec/tests/PlanBuilderTest.cpp

```diff
@@ -148,8 +158,22 @@ TEST_F(PlanBuilderTest, windowFrame) {
                "window1(c) over (partition by a order by b rows between current row and unbounded following) as d7",
                "window1(c) over (partition by a order by b range between current row and unbounded following) as d8",
                "window1(c) over (partition by a order by b rows between unbounded preceding and unbounded following) as d9",
-               "window1(c) over (partition by a order by b rows between unbounded preceding and unbounded following) as d10"})
-          .planNode());
+               "window1(c) over (partition by a order by b range between unbounded preceding and unbounded following) as d10"})
+          .planNode()
+          ->toString(true, false),
```

Comment:
Should we move these tests to PlanNodeToStringTest?

Reply:
These tests were originally expected to test that different variations of frame with the same the partition by and order by clauses all work in the same WindowNode. On account of that I felt it is a better fit here than in PlanNodeToString test. wdyt ?

Reply:
Thanks for clarifying. Have fixed this.